hr_holidays_compute_days/models/hr_holidays_status.py

```diff
@@ -1,20 +1,29 @@
 # -*- coding: utf-8 -*-
-# ©  2015 iDT LABS (http://www.@idtlabs.sl)
+# Copyright 2015 iDT LABS (http://www.@idtlabs.sl)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from odoo import fields, models
 
 
 class HrHolidaysStatus(models.Model):
     _inherit = 'hr.holidays.status'
 
     exclude_rest_days = fields.Boolean(
-        'Exclude Rest Days',
+        string='Exclude Rest Days',
+        default=True,
         help="If enabled, the employee's day off is skipped in leave days "
              "calculation.",
     )
     exclude_public_holidays = fields.Boolean(
-        'Exclude Public Holidays',
+        string='Exclude Public Holidays',
```

Comment:
I am being picky, but for those 3 fields you should not use the string attribute as the technical name is the same as the label.

Reply:
You'd loose control over capitalization of the words.

Reply:
Well, in this case it would be the same, but for this I was looking the "better explicit than implicit".